plugins/ros-pac-man/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1",
+    "information": {
+        "name": "routeros-packet-manager",
```

Comment:
last thing to do is to rename the directory (`ros-pac-man` -> `routeros-packet-manager`)

other than that, everything looks fine